src/vs/editor/browser/widget/diffEditorWidget.ts

```diff
@@ -1611,7 +1611,7 @@ const DECORATIONS = {
 	}),
 	lineInsertWithSign: ModelDecorationOptions.register({
 		className: 'line-insert',
-		linesDecorationsClassName: 'insert-sign',
+		linesDecorationsClassName: 'insert-sign.codicon-add',
```

Comment:
These strings make it into class="...", so I believe class names should be space separated. Today it might work because we validate the string and replace the `.` with a space. I think this should be `"insert-sign codicon codicon-add"`